src/modules/project/pages1/projectDashboard/views/promote/ProjectDashboardPromote.tsx

```diff
@@ -319,10 +319,12 @@ export const ProjectDashboardPromote = () => {
           const user = ambassadorRow.node?.user
           if (!user) return null
           return (
-            <HStack spacing={2}>
-              <UserAvatar user={user} size="xs" />
-              <Body size="sm">{user.username || t('Unnamed User')}</Body>
-            </HStack>
+            <Link href={`/hero/${user.heroId}`} isExternal _hover={{ textDecoration: 'none' }}>
+              <HStack spacing={2}>
```

Comment:
I've started using more of this pattern, it's not necessary but makes it cleaner:

```suggestion
              <HStack as={Link} href={`/hero/${user.heroId}`} isExternal spacing={2}>
```